README.md

```diff
@@ -22,12 +22,38 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
+Note that this process is for Windows. Use equivalent actions for other operating systems. This guide is also meant to be understandable by users with little to no knowledge of Git.
+
+(credit to Hyperion-21)
+
+**SETUP**
+1. Download FFmpeg at https://www.gyan.dev/ffmpeg/builds/ffmpeg-release-full.7z
+2. Extract the .zip onto desktop, rename it to FFmpeg
+3. Move the folder into `C:\Program Files`, anywhere works but there is convienient
+4. Go to `Control Panel\System and Security\System` and click on `Advanced system settings`
+5. Go to the `Advanced` tab. Click on `Enviornment Variables...`
+6. Click on the listing that says `PATH` and click edit. 
+- On win10, you should see a list. Click `New` then type in `C:\Program Files\FFmpeg\bin`
+- On win7, you should see a window that says `Edit User Variable.` In the text field that says "Variable value:" you want to type at the end `;C:\Program Files\FFmpeg\bin`. The semicolon is for seperating the file path from the other file paths in that field.
+7. FFmpeg should now be installed. To test if you did this correctly, run `cmd.exe` as administrator and type `ffmpeg -version` and it should spit at you several lines of code.
+8. Install `Node.js`, `Yarn`, and `GitHub CLI`. All three of these softwares use setup wizards, so installation should be easy. If you don't know what setting to put in, use the default. Also make sure you have `Java` and `Git`, but both are very commom afaik. Run `cmd.exe` if you closed it earlier and run `java -version` and `git --version` to test if you already have them installed or not.
+9. `cd C:\Program Files (x86)\GitHub CLI` then `gh` in `cmd.exe`. Login through that, selecting HTTP over SSH.
+10. `cd %USERPROFILE%` to return to your user folder.
+11. `gh repo clone tobspr/shapez.io` to make sure the game's code is on your system.
+12. `git remote add upstream https://github.com/tobspr/shapez.io.git` for resetting branches to the current and official build on GitHub.
+13. Set up branches for development. If you just want to build and not make anything, skip this step. On your fork, you should create a new branch. Each unique pull request should have its own branch. Reserve the `master` branch to being level with `tobspr/master`. Run `git reset --hard upstream/(username)/master && git push --force` to reset your master to be level with tob's. Make sure all of your current work is on the branch you created.
+
+**EXECUTION**
+1. Start `cmd.exe`
+2. If you aren't already on `C:\Users\[Your user]`, type `cd %USERPROFILE%`
+3. `cd shapez.io`
+4. See below for switching builds. If you wish to perform any of those commands, run them at this step.
+5. `yarn & cd gulp & yarn & yarn gulp`. This will take a while, but afterwards shapez.io should be running in a browser tab. Note that if both `yarn` commands respond with `Already up-to-date`, if the future you only have to run `cd gulp & yarn gulp`, but only do this if you know you are up to date.
```

Comment:
Might be better to do
```suggestion
5. `yarn && cd gulp && yarn && yarn gulp`. This will take a while, but afterwards shapez.io should be running in a browser tab (there may be an error at first, but it should quickly go away).
- Note that if both `yarn` commands respond with `Already up-to-date`, you only have to run `cd gulp && yarn gulp` in the future, as long as there are no changes to the dependencies.
- Also note that `cd gulp` takes you from the `shapez.io` directory to the `gulp` directory. If you are already in the `gulp` directory (current directory is by the prompt), skip the `cd gulp` command.
```
(double ampersands to only continue if command succeeds)